app/components/Footer/index.jsx

```diff
@@ -9,37 +9,22 @@ export default class Footer extends Component {
     return <footer id="Footer">
       <div className="footer-content">
         <div className="footer-links">
-          <ul>
-            <h4>Explore</h4>
-            <li><Link to="/search/">Reports</Link></li>
-            <li><Link to="/visualize">Viz Builder</Link></li>
-            <li><Link to="/map/">Maps</Link></li>
-            <li><Link to="/cart/">Data Cart</Link></li>
-          </ul>
-          <ul>
-            <h4>Sources</h4>
-            <li><Link to="/about/datasets/">Data Sources</Link></li>
-            <li><Link to="/about/api/">API</Link></li>
-            <li><Link to="/about/classifications/">Classifications</Link></li>
-            <li><a href="mailto:hello@datausa.io">Contact Us</a></li>
-          </ul>
-          <ul>
-            <h4>About</h4>
-            <li><Link to="/about/">Background</Link></li>
-            <li><Link to="/about/press/">In the Press</Link></li>
-            <li><Link to="/about/team/">Team</Link></li>
-            <li><Link to="/about/glossary/">Glossary</Link></li>
-            <li><Link to="/about/usage/">Terms of Use</Link></li>
-          </ul>
+          <img src="/images/logo_sm.png" alt="Data USA" />
+          <p>Explore, map, compare, and download U.S. data</p>
+          <div className="footer-column">
+            <div><h4><Link to="/">Home</Link></h4></div>
+            <div><h4><Link to="/search/">Reports</Link></h4></div>
+            <div><h4><Link to="/visualize">VizBuilder</Link></h4></div>
+            <div><h4><Link to="/map/">Maps</Link></h4></div>
+            <div><h4><Link to="/about/background">About</Link></h4></div>
```

Comment:
The `<h4>` wrapper element shouldn't be used for these links, we can just remove them entirely (and update any CSS referencing them). The `<h*>` elements should really just be used for titles that denote different sections on the page (the term comes from Heading 1, Heading 2, etc in a word processor, and screen readers use them extensively).